packages/Utils/src/DTK_Core.hpp

```diff
@@ -0,0 +1,28 @@
+/****************************************************************************
+ * Copyright (c) 2012-2017 by the DataTransferKit authors                   *
```

Comment:
The only thing that is not included in `DTK_C_API.h` are the types and the version/hash. So I'm not sure what you mean here.